require.js

```diff
@@ -592,6 +593,33 @@
                             Promise.all(promises)) : null;
             }, function (error) {
                 getModuleDescriptor(topId).error = error;
+            }).then(function (value) {
+                var module = getModuleDescriptor(topId);
+                if (module.location && (endsWith(module.location, ".meta") || endsWith(module.location, ".mjson"))) {
+                    if (typeof module.exports !== "object" && typeof module.text === "string") {
+                        if (Require.delegate && typeof Require.delegate.compileMJSONFile === "function") {
+                            return Require.delegate.compileMJSONFile(
```

Comment:
If `mr` now knows how to compile MJSON files, why do we need a delegate?

Reply:
It doesn't, I just added a way to parse serialization inside mr, which can be delegated later, but this is invoking code that instantiate a deserializer object, which live in montage only.

Reply:
That begs the question: should `mr` even be aware of `mjson` files at all, instead of letting montage add its own loader and compiler? If we gradually move montage-specific features out of mr and into montage we'll end up with a more generic module loader that would be easier to replace in the future.

Reply:
mjson handing is now in montage